### ISSUE TITLE
Possible clarification needed for Mac install instructions (XCode, npm version)

### ISSUE THREAD

**Issue by jeffbaumes** [girder/girder#1785]
See discussion on https://github.com/memex-explorer/image_space/pull/189

@zachmullen do you agree that these should be addressed?

**Comment by zachmullen** [comment]
Yep, we should look into it and make sure our macos instructions work from a clean system.
